### PR TITLE
Aside

### LINKED ISSUE #14: aside
# Purpose

I believe it would be stylistically valuable to include an `<aside>` into our home page. This aside would be updated every time a document is dropped/uploaded and would have some feature to allow removing documents from being compared.

# Benefit

We should initially be focusing on comparing only two documents at once, however we should also have some consideration into how we are going to handle comparing multiple documents at once. This aside would be easily able to manage both in a very clean way.

- two documents at once: set a limit of two documents to the input but set up the aside to dynamically grow larger than that.
- multiple documents: simply remove the limit on documents and the dynamic aside will also be able to handle it

This will also provide a clean and simple way to view and access all uploaded documents. It could possibly also be extended to handle post comparison features, like going between documents to see the matching sections or displaying the documents and the percentage match in a simple table.

# Immediate Recommendation

Implement a simple aside in the layouts directory of the templates. How this will be extended can be considered more thoroughly and applied at a later date.

### PR DESCRIPTION
**Pull request by Elijah017**
The aside has been completed in a functional sense. There is still room for UI/X design changes and improvements, though that can be investigated further in the future.

### CODE REVIEW

File: app/static/js/home.js [hunk 1]
```diff
@@ -1,24 +1,79 @@
...
+}
+
+function createListItem(name) {
+    let delIcon = $('<i>')
+        .addClass('bi')
+        .addClass('bi-trash3')
+        .click({itemId: fileId}, deleteListItem);
+    
+    let para = $('<p>')
+        .text(name);
```
Comment (jbwod) [inline app/static/js/home.js:40]: `let para = $('<p>')
        .text(name)
        .attr('title', name);`
        
  We can add a simple tooltip here that might work well.

**Review by Lu0922 — APPROVED** [review]
aside file list looks great

File: app/static/js/home.js (hunk 1 above)
Comment (Elijah017) [reply app/static/js/home.js:40]: I have added tooltips now and they are working

**Review by loklokyx — APPROVED** [review]
The tooltip works good now, thank you!

**Pull request merged**